lib/main.js

```diff
@@ -13,6 +13,17 @@ let tslintDef;
 let requireResolve;
 const idleCallbacks = new Set();
 
+async function statAsync(pathname) {
```

Comment:
Fixed in https://github.com/AtomLinter/linter-tslint/pull/157/commits/94ad4d1871edd5b6b3174c8af1d3e9c675ec6e56